### DISCUSSION TITLE
On what ship did the Pilgrims travel to North America in 1620?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#80] [Galaxy Trivia] [answered]
**Hint:** May 🌼

**Comment by Tynab** [comment, marked as answer]
The Pilgrims traveled to North America on a ship called the Mayflower. The Mayflower was a small ship of around 100 feet long and 25 feet wide, and it carried 102 passengers and crew on its journey across the Atlantic from England to what is now known as Massachusetts. The Pilgrims were seeking religious freedom and a new home where they could practice their religion without persecution. They landed at what is now Plymouth, Massachusetts, and established the Plymouth Colony, which became one of the earliest successful colonies in the New World.
